kmod/src/server.c

```diff
@@ -3881,6 +3883,7 @@ static void scoutfs_server_worker(struct work_struct *work)
 	scoutfs_alloc_init(&server->alloc,
 			   &super->server_meta_avail[server->other_ind ^ 1],
 			   &super->server_meta_freed[server->other_ind ^ 1]);
+	alloc_init=true;
```

Comment:
Spaces around = (and a trivial thoes typo in the message that I can't comment on specifically \o/)